pkg/logentry/stages/template.go

```diff
@@ -32,6 +34,11 @@ var (
 		"TrimPrefix": strings.TrimPrefix,
 		"TrimSuffix": strings.TrimSuffix,
 		"TrimSpace":  strings.TrimSpace,
+		"Sha256": func(salt string, s string) string {
+			hasher := sha256.New()
```

Comment:
I think this will be better https://golang.org/pkg/crypto/sha256/#Sum256 since you don’t reuse the hasher.

The reason is shorter code but also less heap allocation for high volume.

Reply:
agreed. changed!